### ISSUE TITLE
Failed to install 4.2.2 from pypi

### ISSUE THREAD

**Issue by jbub** [campaignmonitor/createsend-python#45]
```
Collecting createsend
  Downloading createsend-4.2.2.tar.gz (160kB)
    100% |████████████████████████████████| 163kB 2.4MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/my/cvndjd3s70z22zmybwh59rvc0000gn/T/pip-build-EzVTCF/createsend/setup.py", line 3, in <module>
        from createsend.createsend import __version__
      File "createsend/__init__.py", line 8, in <module>
        from createsend.createsend import *
      File "createsend/createsend.py", line 12, in <module>
        from createsend.utils import VerifiedHTTPSConnection, json_to_py, get_faker
    ImportError: No module named utils
```

**Comment by kamusin** [comment]
Same problem here

**Comment by SpainTrain** [comment]
Silver lining: it probably means that https://github.com/campaignmonitor/createsend-python/issues/34 can be closed?

**Comment by phg49389** [comment]
I have the same error, as well.

**Comment by djstein** [comment]
problem is located here for python3 users: https://github.com/campaignmonitor/createsend-python/blob/master/createsend/__init__.py

**Comment by djstein** [comment]
fixed this on pull request #46.

**Comment by djstein** [comment]
While the pull request is being made if you would like to use the version I have been working on feel free too: https://github.com/getbento/createsend-python

Simply add this to your requirements.py:
git+git://github.com/getbento/createsend-python.git

**Comment by djstein** [comment]
I have updated this again via this pull request #47 thus finally solving the import issues at pip install. Please read the explanation there. Also @jdennes Campaign Monitor has emailed me back saying they have contacted the engineering team.

**Comment by jaraco** [comment]
This issue affected me, although I got a slightly different error:

```
Collecting createsend
  Downloading createsend-4.2.2.tar.gz (160kB)
    100% |████████████████████████████████| 163kB 1.0MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/pip-build-7c99r2qj/createsend/setup.py", line 3, in <module>
        from createsend.createsend import __version__
      File "/private/var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/pip-build-7c99r2qj/createsend/createsend/__init__.py", line 20, in <module>
        from createsend.version import __version__
    ModuleNotFoundError: No module named 'createsend.version'
```

Pinning to 4.2.1 seems to work around the issue.

**Comment by jaraco** [comment]
I spoke too soon. Installing 4.2.1 from PyPI also fails if six is not already installed.

```
Collecting createsend!=4.2.2 (from -r requirements.txt (line 14))
  Downloading createsend-4.2.1.tar.gz (159kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-0phub523/createsend/setup.py", line 5, in <module>
        from createsend.createsend import __version__
      File "/tmp/pip-build-0phub523/createsend/createsend/__init__.py", line 1, in <module>
        ﻿from .createsend import *
      File "/tmp/pip-build-0phub523/createsend/createsend/createsend.py", line 6, in <module>
        from six import BytesIO
    ModuleNotFoundError: No module named 'six'
```

**Comment by xfxf** [comment]
I just noticed in your git commit history this was fixed, and 4.2.3 and then 4.2.4 was meant to be released.  This hasn't been pushed to pypi - could it please?

**Comment by theholy7** [comment]
Please push this fix so we can `pip install createsend` without workarounds

**Comment by djstein** [comment]
@xfxf @theholy7 @jaraco yep this is fixed with version 4.2.3 and 4.2.4. if you want to put pressure on the campaign monitor team to release it please do. for now if you still need it please grab from https://github.com/getbento/createsend-python

to install please add `git+git://github.com/getbento/createsend-python.git` to your `requirements.txt`

**Comment by katharosada** [comment]
Fixed in d1d545f6e9f85eb1063c64fa2d5d383f844bfe9d and pushed in version 4.2.6.
